src/pyflex/window_selector.py

```diff
@@ -186,6 +189,18 @@ def default(self, obj):
             except TypeError:
                 filename.write(j.encode())
 
+    def write_text(self, filename):
+        """
+        Write plain text file for the window output
+        """
```

Comment:
I have added documents. 

By "add a short example", you mean something like this?
```
    def write_text(self, filename):                                             
        """                                                                     
        Write windows to a plain text file. For example, after                  
        you selecting windows, you want to write them out and                   
        save as text files, you can call:                                       
        ws.write_text("window.txt")                                             
                                                                                
        :param filename: Name to write to.                                      
        :type filename: str                                                     
        """  
```